### ISSUE TITLE
Its time to continue this repo.

### ISSUE THREAD

**Issue by Colonizor48** [cx88/diepssect#5]
We are so close to cracking the diep.io server code. Lets finish the job. I dont have discord. But if i did i would make a new server for this.

**Comment by Colonizor48** [comment]
Depending on how easy the raw source code is to modify we could remake arras.io but with the original diep physics.

**Comment by abcxff** [comment]
Sounds cool! Some friends and I are working on a more complete and updated version of this repo with precise information on memory, wasm, physics, constants and packets. When/if you make a discord, my tag is ABC#3752, and you can find me in the [spade squad discord](http://spade-squad.com/)

**Comment by Colonizor48** [comment]
Unfortunately I'm not allowed to use discord. But i will help regardless. I can link you to another repo with another person who is decompiling diep. With an archive of repos i belive will prove to be useful.

**Comment by Colonizor48** [comment]
https://github.com/rafern/diep.io-wasm-re. Go to issues to find the archive of repos i mentioned

**Comment by Colonizor48** [comment]
https://static.diep.io/build_e99021e7fccb3f654e220f67668379afb0f4cc77.wasm.js You could probably find this by yourself. But here is the webasembly for the latest version of diep.

**Comment by abcxff** [comment]
Me and my friends have been working on the game for the past 8 months (and some 1-3 years) lol, if there is any other way we can communicate with you, let us know - we love finding more interested people

**Comment by abcxff** [comment]
Hue works with us, though he is no longer as active - also CX we have in contact as well as Pola (who made the op bots). Though all information on the game is basically outdated especially since there have been no public unshufflers released

**Comment by Colonizor48** [comment]
Yes there is a way actually. 2 main ways. 1. via steam messenger. I could create a steam group for this project if needed. 2. Via github discussions. Github has a discussion feature now. You just have to enable it in settings. You could enable that for the repo and i could communicate that way. In fact that might actually be better then discord for some uses because it can be more easly intigrated into workflow as its part of github.

**Comment by abcxff** [comment]
Oh ok, let me try this out rq and we can discuss

**Comment by Colonizor48** [comment]
Also could you get into contact with CX and inform him that arras.io kinda has to be open source? Because it was forked from neph's arrasio which had the GPLv3 license. Which requires that any derrivitives are open source. And modern arras.io is a derrivitive of neph's. And in adition to making private servers so much easier(the template right now is very out of date). But also because i dont want arrasio to run into legal issues down the line.

**Comment by abcxff** [comment]
They are working on rewriting the game and neph has given them permission

**Comment by Colonizor48** [comment]
Allright. Still could you atleast ask him to update the private server template with the game? There are a lot of things i wana mess around with.

**Comment by abcxff** [comment]
https://github.com/ABCxFF/diep-leftovers/discussions/1 here's a discussion link, let me know if you have trouble making it work (we should stop commenting on here cuz its spamming diepssect's discord) - we can talk there
